api/build.gradle

```diff
@@ -15,6 +15,8 @@ dependencies {
             libraries.jsr305,
             libraries.errorprone.annotations
     implementation libraries.guava
+    implementation "com.google.protobuf:protobuf-javalite:3.19.2"
```

Comment:
We do not want api/ to depend on protobuf. I think we'll probably want to extract the message class from the MethodDescriptor's request marshaller via https://grpc.github.io/grpc-java/javadoc/io/grpc/MethodDescriptor.ReflectableMarshaller.html and drop the dep on protobuf.

Reply:
I think I misunderstood the purpose of it depending on protobuf here. Basing it on the request message directly could be fine. But the "grab the first generic" stuff is something we'd try to avoid.